src/parser/exclusion_parser.cpp

```diff
@@ -137,6 +149,9 @@ filter_spec_container parse_filters(parameter::vector &filter_array, base_sectio
 
             info.add_loaded(id);
             add_addresses_to_info(addresses, info);
+        } catch (const unsupported_operator_version &e) {
+            DDWAF_WARN("Skipping filter '{}': {}", id, e.what());
+            info.add_skipped(id);
```

Comment:
just to keep in mind that if we're adding new diagnostics, the telemetry rfc should be updated

Reply:
Indeed, I am adding these to the new telemetry RFC.